### DISCUSSION TITLE
Full REST API documentation

### DISCUSSION THREAD

**Discussion by pczern** [supabase/supabase#30213] [Questions] [answered]
I am trying to build an app in golang and am calling the Supabase API for authentication. When sending magic link or password recovery I didn't find the `redirect_to` query param in https://github.com/supabase/auth/blob/master/openapi.yaml like the Supabase clients have. Is there a full up to date rest api documentation of Supabase or am I calling the wrong endpoints for magic link and password recovery and need another for `redirect_to`?

(Also I would love to see an official supported Go client for Supabase, as HTMX and Go can be far more powerful than Frontend frameworks when used well)

Update: openapi.yaml seems the most up to date

**Comment by GaryAustin1** [comment, marked as answer]
More on REST auth calls in the readme docs at: https://github.com/supabase/auth?tab=readme-ov-file#endpoints

Also there is this work that might be useful:  https://github.com/supabase-community/auth-go

**Reply by pczern** [reply]
where can i set the `SITE_URL` in supabase? I assume /recover and /magiclink will link there?

**Reply by GaryAustin1** [reply]
![image](https://github.com/user-attachments/assets/a86b7fbb-56cc-4b7d-bdf6-fef360172aac)

**Reply by pczern** [reply]
great! Thanks!
